### DISCUSSION TITLE
what can i do to make it possible to use juicefs on foundationDB?

### DISCUSSION THREAD

**Discussion by Theseaway** [juicedata/juicefs#2162] [Q&A]
I  see the list of supported metadata engines on your official website, I want to know what can i do if I need to use juicefs on foundationDB? Or how the juicefs can support other engines? where can i find the code in this repository?

**Comment by Theseaway** [comment]
I find the meta directory in pkg, I am trying to understand how it works to support metadata engine.

**Reply by solracsf** [reply]
There is already a draft PR here https://github.com/juicedata/juicefs/pull/644
